bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/handler/NukiSmartLockHandler.java

```diff
@@ -220,6 +221,12 @@ private void handleCommandRefreshType(ChannelUID channelUID, Command command) {
                     updateState(channelUID, bridgeLockStateResponse.isBatteryCritical() ? OnOffType.ON : OnOffType.OFF);
                 }
                 break;
+            case NukiBindingConstants.CHANNEL_SMARTLOCK_DOOR_STATE:
+                bridgeLockStateResponse = nukiHttpClient.getBridgeLockState(nukiId);
+                if (handleResponse(bridgeLockStateResponse, channelUID.getAsString(), command.toString())) {
+                    updateState(channelUID, new DecimalType(bridgeLockStateResponse.getDoorsensorState()));
```

Comment:
See other comment about channel item-type:
```suggestion
                    updateState(channelUID, new StringType(bridgeLockStateResponse.getDoorsensorState()));
```